js/src/api/format/input.js

```diff
@@ -127,6 +127,18 @@ export function inNumber16 (number) {
   return inHex(bn.toString(16));
 }
 
+export function inOptionsCondition (condition) {
+  if (condition) {
+    if (condition.block) {
+      condition.block = condition.block ? inNumber10(condition.block) : null;
+    } else if (condition.time) {
+      condition.time = inNumber10(Math.floor(condition.time.getTime() / 1000));
```

Comment:
Is there any other RPC calls with time/date involved ? Might be useful to have an `inTime` formatter to make it DRY